gui/autobutcher.lua

```diff
@@ -402,9 +409,25 @@ function WatchList:init()
                         auto_width=true,
                         on_activate=self:callback('onSetRow'),
                     },
+                    widgets.HotkeyLabel{
+                        view_id='butcher_all',
+                        frame={t=5, l=0},
```

Comment:
When we first released this tool, the labels were unaligned in the footer and we got a lot of complaints about how messy it looked. I suggest reorganizing the labels so it looks like this:

```
f: f kids        F: F adults        w: Toggle watch  r: Set row targets to 0
m: m kids        M: M adults        x: Delete row    R: Set row targets to N
b: Butcher race  B: Unbutcher race  e: Butcher all   E: Unbutcher all
A: Autobutcher is Enabled           W: Autowatch is Enabled
```